api/users/views.py

```diff
@@ -288,25 +287,23 @@ class UserNodes(JSONAPIBaseView, generics.ListAPIView, UserMixin, ODMFilterMixin
     view_category = 'users'
     view_name = 'user-nodes'
 
+    ordering = ('-date_modified')
```

Comment:
Just realized: this evaluates to a string, when I believe what we want is a tuple. It's a quirk of Python that you need to add a trailing comma to make this a tuple, since otherwise the Python interpreter assumes you meant the parentheses to be used simply for grouping.

```
$ python2
Python 2.7.9 (default, Jan  4 2015, 20:46:16) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.56)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> ('foo')
'foo'
>>> ('foo',)
('foo',)
>>>
```

I noticed when comparing with `NodeList` under https://github.com/caneruguz/osf.io/pull/17#issuecomment-180575710. Not sure what the real-world effect would be in this case. Maybe (probably?) our ODM framework is smart enough to handle a string here, in which case we should drop the parens. :)